geode-docs/developing/transactions/working_with_transactions.html.md.erb

```diff
@@ -76,37 +76,48 @@ r.put("stringBuf", s);
 cTxMgr.commit();
 ```
 
-## Transactions and Functions
+## <a id="concept_ihn_zmt_vk" class="no-quick-link"></a>Transactions and Functions
 
-You can run a function from inside a transaction and you can nest a transaction within a function, as long as your combination of functions and transactions does not result in nested transactions. See [Function Execution](../function_exec/chapter_overview.html) for more about functions.
+A function may contain transactions, and a transaction may contain functions, as long as you observe these restrictions:
 
-A single transaction may contain multiple functions.
+  - **Your combination of functions and transactions must not result in nested transactions.**
 
-If you are suspending and resuming a transaction with multiple function calls, all functions in the transaction must execute on the same member.
+  - **When a function contains multiple transactions,** the transactions must be sequential, not nested.
```

Comment:
I think this is the same as the bullet above

Reply:
You're right, it's redundant. Will delete.